subsys/nrf_security/configs/psa_unsupported_ciphers_check.h

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ * Copyright (c) since 2020 Oberon microsystems AG
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Currently Unsupported Algorithms */
+
+#if defined(CONFIG_PSA_WANT_ALG_SHA_512_224) && !defined(CONFIG_PSA_ACCEL_SHA_512_224)
+#error "No crypto implementation for SHA-512-224"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHA_512_256) && !defined(CONFIG_PSA_ACCEL_SHA_512_256)
+#error "No crypto implementation for SHA-512-256"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHA3_224) && !defined(CONFIG_PSA_ACCEL_SHA3_224)
+#error "No crypto implementation for SHA3-224"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHA3_256) && !defined(CONFIG_PSA_ACCEL_SHA3_256)
+#error "No crypto implementation for SHA3-256"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHA3_384) && !defined(CONFIG_PSA_ACCEL_SHA3_384)
+#error "No crypto implementation for SHA3-384"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHA3_512) && !defined(CONFIG_PSA_ACCEL_SHA3_512)
+#error "No crypto implementation for SHA3-512"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_SHAKE256_512) && !defined(CONFIG_PSA_ACCEL_SHAKE256_512)
+#error "No crypto implementation for SHAKE-256"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_MD5) && !defined(CONFIG_PSA_ACCEL_MD5)
+#error "No crypto implementation for MD5"
+#endif
+#if defined(CONFIG_PSA_WANT_ALG_RIPEMD160) && !defined(CONFIG_PSA_ACCEL_RIPEMD160)
+#error "No crypto implementation for RIPEMD160"
+#endif
+
+#if defined(CONFIG_PSA_WANT_KEY_TYPE_AES) && defined(CONFIG_PSA_WANT_ALG_CFB)
```

Comment:
```suggestion
#if defined(CONFIG_PSA_WANT_ALG_CFB)
```

Checking for KEY_TYPE_AES seems unnecessary. Unless it is possible to enable KEY_SIZE_128 without enabling KEY_TYPE_AES?